app/views/extended_metadata_types/_extended_metadata_type_table_row.html.erb

```diff
@@ -0,0 +1,19 @@
+<tr class="<%= 'warning' if extended_metadata_type.disabled_but_used? %>">
```

Comment:
maybe also add a css to turn the text color in the row to **grey** when EM is disabled.  ( optional )